docs/theme/home.html

```diff
@@ -16,7 +16,7 @@
         Python, JavaScript, and Ruby) with a common output format, and developer
         tools that consume this format.
 
-        <p>srclib originated inside <a target="_blank" href="https://sourcegraph.com">Sourcegraph</a>, where it powers intelligent code search over hundreds of thousands of projects.
+        <p>srclib originated inside <a target="_blank" href="https://www.sourcegraph.com">Sourcegraph</a>, where it powers intelligent code search over hundreds of thousands of projects.
```

Comment:
fixed, but at some point we will probably want to move them all to www, simply to manage cookie leaks and scale out other non-www services, but that's a problem for another day.